shared/folders/files/render.desktop.js

```diff
@@ -263,14 +263,22 @@ class FilesRender extends Component<Props> {
               bottom: 16,
             }}
           >
-            {this.props.users.map(u => (
-              <Box key={u.username} style={{height: 32, width: 28}}>
-                <Avatar username={u.username} size={32} borderColor={globalColors.white} />
-              </Box>
-            ))}
+            {!this.props.isTeam &&
+              this.props.users.map(u => (
+                <Box key={u.username} style={{height: 32, width: 28}}>
+                  <Avatar username={u.username} size={32} borderColor={globalColors.white} />
+                </Box>
+              ))}
+            {this.props.isTeam &&
+              <Avatar
+                teamname={this.props.users.length ? this.props.users[0].username : 'unknown'}
```

Comment:
should pass isTeam if we know its a team
don't use 'unknown' as that could be an actual team